pkg/kdump/kdump-view.jsx

```diff
@@ -127,7 +127,7 @@ const KdumpSettingsModal = ({ settings, initialTarget, handleSave }) => {
                         // avoid bad summary like "systemd job RestartUnit ["kdump.service","replace"] failed with result failed"
                         // if we have a more concrete journal
                         error.message = _("Unable to save settings");
-                        error.details = <CodeBlockCode>{ error.details }</CodeBlockCode>;
+                        error.details = <CodeBlockCode>{ error.details.replaceAll("kdump: ", "") }</CodeBlockCode>;
```

Comment:
```suggestion
                        error.details = <CodeBlockCode>{ error.details.replaceAll(/^kdump: /, "") }</CodeBlockCode>;
```

Would this be better to make sure `kdump: ` a regular expression and have it match the beginning of lines? (It _probably_ doesn't matter, though. And I didn't check to see if this actually works. :sweat_smile:)

Reply:
Yes, this is a bit sloppy as it removes one extraneous `kdump: `. in the middle of the line.

Reply:
right, but in some more effort and now it doesn't strip extranous `kdump: `.

![Screenshot from 2023-03-29 10-06-28](https://user-images.githubusercontent.com/67428/228468905-9fabb624-0646-40d6-bf69-8bd0f0f33ee0.png)